public/activity.html

```
@@ -190,405 +173,285 @@ <h3 class="font-bold text-lg mb-4 flex items-center">
-          </div>
-
-          <!-- Sessions Card -->
-          <div id="sessions-card" class="rounded-2xl overflow-hidden bg-white dark:bg-gray-800 border border-gray-200 dark:border-gray-700 shadow-md">
-            <div class="px-6 py-4 border-b border-gray-200 dark:border-gray-700 flex items-center justify-between bg-gradient-to-r from-teal-50 to-cyan-50 dark:from-teal-900/20 dark:to-cyan-900/20">
-              <h2 class="font-bold text-gray-800 dark:text-gray-200 flex items-center">
-                <i class="fas fa-calendar mr-2 text-teal-600 dark:text-teal-400"></i>Sessions
+    <!-- Tab Bar -->
+    <div class="bg-white dark:bg-gray-900 border-b border-gray-200 dark:border-gray-700 sticky top-0 z-20 shadow-sm">
+      <div class="max-w-[90rem] mx-auto px-4 md:px-6">
+        <div class="flex gap-0 overflow-x-auto" id="tab-bar" role="tablist">
+          <button class="tab-btn active" data-tab="overview" onclick="switchTab('overview')" role="tab">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)